custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -131,7 +131,7 @@ def monitor(self, kwargs, predictions, predict_time_ms):
 
     def predict(self, **kwargs):
```

Comment:
this needs a type-hint with return value

Reply:
Done